app-backend/build.sbt

```diff
@@ -33,6 +33,29 @@ lazy val commonSettings = Seq(
   addCompilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
 )
 
+// Create a new MergeStrategy for aop.xml files
+val aopMerge = new sbtassembly.MergeStrategy {
```

Comment:
This was necessary to get the Kamon plugins (Akka-HTTP, StatsD) working when invoked from an assembly JAR.